jax/_src/prng.py

```diff
@@ -580,7 +580,7 @@ def threefry_random_bits(key: jnp.ndarray, bit_width, shape):
 
 def _rbg_seed(seed: int) -> jnp.ndarray:
   halfkey = threefry_seed(seed)
-  return jnp.concatenate([halfkey, halfkey])
+  return jnp.tile(halfkey, 2)
```

Comment:
concatenate repeat axis 0, while tile repeat axis - 1. Are you sure halfkey is 1d array?

Reply:
Yes, this seems to be the case:
https://github.com/google/jax/blob/3bfa6af2c81ac3c08b05d49b70c1bf01f09d28c5/jax/_src/prng.py#L246-L256

Reply:
I think this will probably cause issues with custom PRNG keys, which don't concatenate like regular arrays. (`jnp.concatenate` checks for this, while `jnp.tile` does not). I'm going to request a review from @froystig, who would probably have recommendations here.

Reply:
Good thinking @jakevdp. Actually, in this case, `tile` is valid use and is already supported! For more on how and why see #8381.